src/System.Management.Automation/engine/ComInterop/ExcepInfo.cs

```diff
@@ -16,7 +16,11 @@ namespace System.Management.Automation.ComInterop
     internal struct ExcepInfo
     {
         private readonly short _wCode;
+#if DEBUG
         private short _wReserved;
+#else
+        private readonly short _wReserved;
+#endif
```

Comment:
It seems the code comes from .Net runtime and we shouldn't change it. 
I guess we can suppress on folder level with EditorConfig file which has a priority.

Reply:
We made changes to this code in #13967

The code comes from: https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ExcepInfo.cs

We should see if these changes can be made at `dotnet/runtime`.

Reply:
#14012 reverts changes